VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseOutlineAdapter.java

```diff
@@ -56,12 +57,15 @@
     private IStorage storage;
     private DownloadListener mDownloadListener;
     private Context context;
+    private Config config;
 
     private boolean currentVideoMode;
     private int numOfTotalUnits;
 
-    public CourseOutlineAdapter(Context context, IDatabase dbStore, IStorage storage, DownloadListener listener) {
+    public CourseOutlineAdapter(Context context, Config config, IDatabase dbStore, IStorage storage,
+                                DownloadListener listener) {
```

Comment:
I am passing `Config` as an argument to `CourseUnitPagerAdapter` too, so it seemed logical to have consistency.